pkgs/top-level/all-packages.nix

```diff
@@ -4661,6 +4661,12 @@ with pkgs;
     extraFonts = true;
   };
 
+  texmacsDarwin = callPackage ../applications/editors/texmacs/darwin.nix {
```

Comment:
How about doing an if else block with texmacs above. I think we do it somewhere else. Something like:

```
texmacs = if stdenv.isDarwin then callPackage ...
                  else callPackage ...;
```